gem/test/phlexing/erb_transformer_test.rb

```diff
@@ -27,42 +27,42 @@ class ErbTransformerTest < Minitest::Spec
       input = %(<div><% "The Next line has text on it" %> More Text</div>  )
       expected = %(<div><erb silent> &quot;The Next line has text on it&quot; </erb> More Text</div>)
 
-      assert_equal expected, ErbTransformer.transform(input)
+      assert_equal expected, ERBTransformer.call(input)
```

Comment:
We could leave this class method as `transform`, but I like designing objects that fit the _callable_ duck type since they’re more versatile. The ERB transformer could be replaced with a String-returning Proc, for example.

Reply:
I'm fine if we use `call`. But I guess we should make all of them callable then

Reply:
Probably, I’m just starting here to keep the surface area small.

Reply:
Fair enough 👍🏼